server/app/routes/reviews/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const router = require('express').Router();
+const mongoose = require('mongoose');
+const Reviews = mongoose.model('Reviews');
+module.exports = router;
+
+// we are probably not using /Reviews page right now (stretch feature), unless
+// we have time to build a Recommendations page/ engine that highlights reviews/
+// movies that might interest a particular User.
+
+router.get('/', (req, res, next) => {
```

Comment:
new fetchAll function added in Reviews factory in addition to existing fetchAllByMovie and fetchAllByUser functions